dev/aws/sso.ts

```diff
@@ -202,6 +202,21 @@ const completionSpec: Fig.Spec = {
         },
       ],
     },
+    {
+      name: "login",
+      description:
+        "Refreshes SSO access tokens and logs in. Required setup using ``aws configure sso``",
```

Comment:
Looks like we have some extra `:
```suggestion
        "Refreshes SSO access tokens and logs in. Required setup using `aws configure sso`",
```

Reply:
I've used other AWS commands as template. It seems like all of them use double backticks